src/js/polyfills/webvtt.js

```diff
@@ -692,6 +692,11 @@ function moveBoxToLinePosition(window, styleBox, containerBox, boxPositions, num
         let percentage = 0; // Lowest possible so the first thing we get is better.
         let bestAxis;
 
+        // Prevent empty boxes from entering recursive overlap correction.
+        if (!b.height || !b.width) {
+            return b;
```

Comment:
It may be better/safer to return `specifiedPosition` since prior to these changes this method always returns a new `BoxPosition` instance.

Reply:
If it's an empty div, why waste any processing? If we're running it 

There are pre-existing instances where we return `b`, ie:

```
if (b.within(containerBox)) {
  return b;
}
```

This works because b is already a `BoxPosition` instance at the time findBestPosition is called the first time.

![image](https://user-images.githubusercontent.com/2903955/94170537-3226b580-fe5e-11ea-9d19-0d07f0395972.png)

Reply:
nothing is being wasted, it's already created on line 691.